content/Views/Shared/NotFound.liquid

```diff
@@ -1,7 +1,20 @@
-<section class="section section--narrow">
-    <div class="section__content">
-        <h2>{{ "Oops.. we couldn&rsquo;t find that page" | t | raw }}</h2>
-        <p>{{ "The page you are looking for may no longer exist. Please use the menu to find the page you are looking for, or:" | t }}</p>
-        <p><a href="{{ Request.PathBase }}" class="btn btn--secondary">{{ "Go back to the homepage" | t }}</a></p>
-    </div>
-</section>
\ No newline at end of file
+{% assign contentItem = Content["alias:404"] %}
+
+{% if contentItem != nil %}
+  {{ contentItem | shape_build_display: "Detail" | shape_render }}
+{% else %}
+    <article>
+        <section class="section">
+            <div class="section__content">
```

Comment:
We've removed the need for `section__content` in favour on `constrain` (https://github.com/EtchUK/Etch.OrchardCore.ThemeBoilerplate/commit/d11afc903406879410c358a440246da9ea81122e).